.github/workflows/reusable_basic.yml

```diff
@@ -318,6 +318,56 @@ jobs:
       run: ${{github.workspace}}/.github/scripts/check_dll_flags.ps1 ${{env.BUILD_DIR}}/src/proxy_lib/${{matrix.build_type}}/umf_proxy.dll
       shell: pwsh
 
+  hwloc-fallback:
```

Comment:
hmm is it required to be in basic tests? can it fail often? perhaps move to nightly?

Reply:
Moved to nightly